docs/index.rst

```diff
@@ -93,12 +93,13 @@ then panel specific classes can be added at the start of each panel.
 
 .. seealso::
 
-    `Bootstrap grid examples <https://getbootstrap.com/docs/4.4/examples/grid/>`_
+    The bootstrap 4 `grid documentation <https://getbootstrap.com/docs/4.0/layout/grid/>`_,
+    and this `grid tutorial <https://www.w3schools.com/bootstrap/bootstrap_grid_system.asp>`_
 
 .. code-block:: rst
 
     .. panels::
-        :container: container-lg pad-bottom-20
+        :container: container-lg pb-3
```

Comment:
This doesn't *always* need `:container:` though, right? I feel like it should have reasonable defaults so users don't have to start writing their own bootstrap classes off the bat

Reply:
No, the defaults are described in the lines above this, and are defined in the code here: https://github.com/executablebooks/sphinx-panels/blob/195e853a214df6234c107d5015b3213070bec42c/sphinx_panels/__init__.py#L34-L36

Note that all classes I use in the documentation are built-in boostrap classes, which are included in the CSS loaded here